source/CommonJobs/CommonJobs.Domain.EvalForm/CalificationItem.cs

```diff
@@ -9,7 +9,7 @@ public class CalificationItem
     {
         public string Key { get; set; }
 
-        public decimal Value { get; set; }
+        public decimal? Value { get; set; }
```

Comment:
I agree that this change is necessary, but looks like a big one, did you check all implications in backend and frontend?